### PR TITLE
Fix #301 #302 - Involved and Vehicles data

### LINKED ISSUE #301: Add the "involved" data to the accident marker
info about the involved people in an accident is available in the: H********InvData.csv file, and should be connected with the specific accident using the identical ID.

### PR DESCRIPTION
**Pull request by omerxx**
#301 & #302 - Data is now pulled on marker click.

### CODE REVIEW

File: static/js/marker.js [hunk 1]
```diff
@@ -154,8 +111,59 @@ var MarkerView = Backbone.View.extend({
         return "/?marker=" + this.model.get("id") + "&" + app.getCurrentUrlParams();
     },
     clickMarker : function() {
+        that = this;
         $.get("/markers/" + this.model.get("id"), function (data) {
-            //alert(data);
+            data = JSON.parse(data);
+
+            var localize_data = function(field,value,dataType) {
```
Comment (danielhers) [inline static/js/marker.js]: Use `function localize_data(field,value,dataType)` rather than using `var`

File: static/js/marker.js [hunk 2]
```diff
@@ -154,8 +111,59 @@ var MarkerView = Backbone.View.extend({
...
+                if (dataType === "invs") {
```
Comment (danielhers) [inline static/js/marker.js]: Use a `switch` statement

File: templates/index.html [hunk 3]
```diff
@@ -272,25 +272,7 @@ <h4 class="title"></h4>
             <p class="mainStreet" style="margin:0"></p>
             <p class="secondaryStreet" style="margin:0"></p>
             <p class="junction" style="margin:0"></p>
-            <p class="involved_type" style="margin:0"></p>
```
Comment (danielhers) [inline templates/index.html]: Why did you remove these? Isn't it better to leave the placeholders so the order can be controlled by editing the HTML and not the code? If the data isn't there, they'll just stay empty.

Comment (omerxx) [inline templates/index.html]: @danielhers Since the deleted fields belong to 'involved' and 'vehicles' data, a marker may contain several blocks of each. Hence the jQuery `.append` and the removal of the above fields

Comment (danielhers) [inline templates/index.html]: I see. Still, perhaps it would make sense to write the HTML as a template,
so that in the code you just fill in the fields and append it. See e.g. the
sidebar entry template.

Comment (omerxx) [inline templates/index.html]: Since @OmerSchechter is going to take this and use it in the new accordion design for the marker I don't think it is required..
